romana-install/roles/romana/install-agent/templates/systemd/romana-policy-agent.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Romana Cloud Native Network Policy Agent Service
+Documentation=https://github.com/romana/core
+
+[Service]
+#User=...
+EnvironmentFile=-{{ romana_etc_dir }}/agent
```

Comment:
The two are going to be merged eventually, but right now `policy-agent` ignores all the environment variables.
So either leave as it is or remove environment.